pyvista/core/filters/poly_data.py

```diff
@@ -1593,28 +1597,32 @@ def decimate(
             volume preservation is disabled and if ``attribute_error``
             is active, these errors can be large.
 
-        attribute_error : bool, default: False
-            Decide whether to include data attributes in the error metric. If
-            ``False``, then only geometric error is used to control the
-            decimation. If ``True``, the following flags are used to specify
-            which attributes are to be included in the error calculation.
+        boundary_constraints: bool, default: False
+            Use the legacy weighting by boundary_edge_length instead of by
+            boundary_edge_length^2 for backwards compatibility
 
-        scalars : bool, default: True
-            If attribute errors are to be included in the metric (i.e.,
-            ``attribute_error`` is ``True``), then these flags control
-            which attributes are to be included in the error
-            calculation.
+        boundary_weight: float, default: 1.0
+           A floating point factor to weigh the boundary quadric constraints
+           by: higher factors further constrain the boundary.
```

Comment:
Include a `.. versionadded:: 0.45` directive
```suggestion
        boundary_weight: float, default: 1.0
           A floating point factor to weigh the boundary quadric constraints
           by: higher factors further constrain the boundary.
           
           .. versionadded:: 0.45
```